scripts/data_collection/iffy_update.py

```diff
@@ -0,0 +1,108 @@
+"""
```

Comment:
Please review this and make sure that I didn't break anything. I tested it a couple of times to see if it works (seems like it does) and if breaking it logs nice notes (seems like it does).

Reply:
Also, please check that what I've written for the input/output is accurate. I am realizing now that the script may not take the path for the existing file. If it is just the output path for where we will save the new file, please make sure to update the docstring, the `argparse` text, and anything else.